### PR TITLE
docs/usage: add information about LOUNGE_HOME variable

### PR DESCRIPTION
**Pull request by williamboman**
/cc @xPaw

### CODE REVIEW

File: _docs/getting_started/usage.md [hunk 1]
```diff
@@ -125,3 +130,12 @@ Example:
 ```
 $ lounge --version
 ```
+
+## Environment variables
+
+The Lounge is also configurable through the use of environment variables. CLI options will always have
```
Comment (xPaw) [inline _docs/getting_started/usage.md]: I think this should be reworded to say that home path is configurable via ENV param. Everything else should be configured via the config.

Comment (williamboman) [inline _docs/getting_started/usage.md]:
```diff
diff --git a/_docs/getting_started/usage.md b/_docs/getting_started/usage.md
index dab5a95..43167a2 100644
--- a/_docs/getting_started/usage.md
+++ b/_docs/getting_started/usage.md
@@ -133,8 +133,7 @@ $ lounge --version
 
 ## Environment variables
 
-The Lounge is also configurable through the use of environment variables. CLI options will always have
-higher precedence than environment variables.
+It is also possible to configure the home path of The Lounge through the following environment variable;
 
 ### `LOUNGE_HOME`
 ```
¯\_(ツ)_/¯?

Comment (xPaw) [inline _docs/getting_started/usage.md]: > It is also possible to configure the home path of The Lounge through the `LOUNGE_HOME` environment variable.

In fact, do we even need to make it a separate section, could it just be left under `--home`?

Comment (williamboman) [inline _docs/getting_started/usage.md]: Like so?

**Comment by astorije** [comment]
👍 Merging this to be in sync with the code, but why did we need this already? I'm thinking that using `--home` is enough but maybe I am overseeing use cases?

**Comment by williamboman** [comment]
@astorije See https://github.com/thelounge/lounge/issues/365#issuecomment-222538947 and https://github.com/thelounge/docker-lounge/issues/9#issuecomment-225557492

**Pull request merged**